src/App.tsx

```diff
@@ -1,26 +1,9 @@
-import React from 'react';
-import logo from './logo.svg';
 import './App.css';
-
+import Navbar from './components/navbar';
+import AddNote from './components/addNote';
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Navbar />;
```

Comment:
This should be:
```
return 
    <>
        <Navbar />
        <AddNote />
    </>
```

Reply:
Refer to this explanation:
https://kinsta.com/knowledgebase/jsx-expressions-must-have-one-parent-element/